envoy_pkg/workspace_patches/istio-proxy/0000-WORKSPACE.patch

```diff
@@ -13,20 +13,20 @@
 # limitations under the License.
 
 diff --git a/WORKSPACE b/WORKSPACE
-index a9c0baa..1b956aa 100644
+index 22b90ef..4b8325e 100644
 --- a/WORKSPACE
 +++ b/WORKSPACE
-@@ -14,11 +14,17 @@
+@@ -14,12 +14,16 @@
```

Comment:
can you make this as a new patch? (i.e. 0001-WORKSPACE.patch), the workspace script is trying patches in descending order, so the old workspace will still get patched correctly.

Reply:
Sounds good. I’ll change this.

Reply:
98cdd5e